src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -72,7 +73,7 @@ protected function formMultipleElements(FieldItemListInterface $items, array &$f
     $parents = $form['#parents'];
 
     $target_type = $this->fieldDefinition->getTargetEntityTypeId();
-    $user_groups = Og::getEntityGroups(\Drupal::currentUser()->getAccount());
+    $user_groups = Og::getEntityGroups(User::load(\Drupal::currentUser()->id()));
```

Comment:
See other comment. This is not consistent in what it returns. Don't always trust type hints from based documentation :)